ctms/app.py

```diff
@@ -255,12 +255,20 @@ def get_bulk_contacts_by_timestamp(
     return CTMSBulkResponse(
         start=start_time,
         end=end_time,
+        after=url_safe_encoded_str,
         limit=limit,
         items=results,
         next=next_url,
     )
 
 
+def updates_helper(value, default):
+    blank_vals = ["", " ", None]
```

Comment:
It seems to me that `blank_vals = ["", None]` should be the list. Was a space (the string `" "`) needed to make this work? If it was, then should any whitespace string be permitted?

Reply:
I can remove this! I wasn't sure if there were any limitations on the cinchy side; a single whitespace would be allowed when this is removed, but would likely break the system. So this was a safety net.

Reply:
Removed the whitespace to just have blank_vals be `["", None]`